package.json

```diff
@@ -1,44 +1,44 @@
-{

```

Comment:
guessing this is a crlf nuke or something?

Reply:
Yeah, most likely